README.md

```diff
@@ -293,7 +299,9 @@ Environment=TARANTOOL_CONSOLE_SOCK=/var/run/tarantool/${app_name}.{instance_name
 To configure replicasets you need to specify replicaset parameters for each instance in replicaset:
 
 * `replicaset_alias` (`string`, optional) - replicaset alias, will be displayed in Web UI;
-* `leader` (`string`, required if `replicaset_alias` specified) - name of leader instance;
```

Comment:
At now does user have a way to mark an instance as a leader? It seems, not.

Reply:
The new way to mark instance as a leader is to specify it first in failover priority.

This configurations are equal:

```yaml
# Old 
replicaset_alias: replicaset-1
leader: replicaset-1-leader

# New
replicaset_alias: replicaset-1
failover_priority:
  - replicaset-1-leader
```

Reply:
It's not obvious for end-user. Your config (aka inventory) needs to be convenient, easy readable, and most of all **obvious**. 

You need to understand that we can deploy via `ansible-cartridge` only **master -> slave** replication configuration and we need to explicit specify, where is the leader.

Reply:
@vasiliy-t @knazarov @rosik we need your opinions.

Reply:
> You need to understand that we can deploy via ansible-cartridge only master -> slave replication configuration

What do you mean, only master-slave? That's wrong. Cartridge as well as vshard always tries to set up full mesh master-master replication.

In general I like Liza's design. The only thing I would suggest is mentioning that the first server in failover_priority list is the leader. And make more cross-references describing all_rw, leadership, failover_priority and failover modes

Reply:
> And make more cross-references describing all_rw, leadership, failover_priority and failover modes

Seems for me it's too difficult and messy.